apps/src/aichat/views/modelCustomization/CompareModelsDialog.tsx

```diff
@@ -1,17 +1,29 @@
 import React, {useState} from 'react';
 
+import {useAppSelector} from '@cdo/apps/util/reduxHooks';
 import AccessibleDialog from '@cdo/apps/templates/AccessibleDialog';
 import {Heading3} from '@cdo/apps/componentLibrary/typography';
 import Button from '@cdo/apps/componentLibrary/button/Button';
 
 import ModelDescriptionPanel from './ModelDescriptionPanel';
 import styles from './compare-models-dialog.module.scss';
+import {ModelDescription} from '../../types';
 
-const CompareModelsDialog: React.FunctionComponent<{onClose: () => void}> = ({
-  onClose,
-}) => {
-  const [chosenModelLeft, setChosenModelLeft] = useState<string>('llama2');
-  const [chosenModelRight, setChosenModelRight] = useState<string>('mistral');
+const CompareModelsDialog: React.FunctionComponent<{
+  onClose: () => void;
+  availableModels: ModelDescription[];
+}> = ({onClose, availableModels}) => {
+  const selectedModelId = useAppSelector(
+    state =>
+      state.aichat.currentAiCustomizations.selectedModel ??
+      availableModels[0].id
+  );
+  const [chosenModelLeft, setChosenModelLeft] =
+    useState<string>(selectedModelId);
+  const [chosenModelRight, setChosenModelRight] = useState<string>(
+    availableModels.find(model => model.id !== selectedModelId)?.id ??
+      selectedModelId
+  );
 
```

Comment:
Nit: unrelated to your changes, but I think the `<div>` on line 33 below is not necessary.